test/routes/reviewRouter.test.js

```diff
@@ -0,0 +1,123 @@
+const expect = require("chai").expect;
+const request = require("supertest");
+
+const app = require("../../app");
+const signToken = require("../../utils/signToken");
+
+describe("GET /review/:reviewId", () => {
+  it("should response with 200 and get review", async () => {
```

Comment:
response 는 명사형이므로 respond로 작문해주시면 좋을 것 같아요. 

```suggestion
  it("should respond with 200 and get review", async () => {
```